CHARTER.md

```diff
@@ -108,9 +108,13 @@ process as standard proposals.
 ## Code of Conduct Enforcement
 
 Removing or disciplining an ART member due to a violation of the [Arcalot Code of
-Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md) is performed
-by the chairs as a pull request to the ART. Only the the chairs vote on these matters,
-with an approval by 75% of the active chairs required for the matter to pass.
+Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md) is initiated
+by any chair as a pull request to the ART. Only the the chairs vote on these matters,
```

Comment:
"the the" chairs... and "these matters" seems unnecessarily convoluted. It's a PR; can we just say "vote on these pull requests"? Or maybe even something like "only chair reviews of these pull requests count" as there's no actual way to prevent anyone else from reviewing the PR, and the chairs are simply allowed to ignore their comments. (I suppose...)

Reply:
I disagree. The pull request is an means to an administrative function. You don't vote on a pull request, you vote on a matter or a proposal. This allows us to avoid the pedantic specifics of saying something like "only chair reviews count" since it is implied already.